### ISSUE TITLE
react-a11y DayPicker warning

### ISSUE THREAD

**Issue by reintroducing** [gpbl/react-day-picker#386]
#I'm working on a project where accessibility is a requirement and I just integrated DayPicker into the fold. I'm using react-a11y to warn me of any possible issues I may run into/omit to make sure I am not forgetting accessibility when developing my components.  I'm seeing this pop up now:

```
DayPicker You have an unlabeled element or control. Add `aria-label` or `aria-labelledby` attribute, or put some text in the element.
```

I can't for the life of me figure out where its coming from or how to resolve it. I'm not using any fancy implementation of DayPicker, just pretty standard stuff:

```jsx
<DayPicker
     selectedDays={this._modifierSelected}
     disabledDays={this._modifierDisabled}
     onDayClick={this._onDaySelected}
     fromMonth={fromMonth}
     toMonth={toMonth}
     initialMonth={initialMonth}
     weekdaysShort={this._weekdayTitles}
     navbarElement={<DatePickerCalendarNavigation />}
     enableOutsideDays
 />
```

Any help would be greatly appreciated.

**Comment by gpbl** [comment]
Thanks for the report! Which version of react-a11y are you using?

**Comment by reintroducing** [comment]
I'm using react-a11y v0.3.3 which seems to be the latest.

**Comment by gpbl** [comment]
I've found the reason of this warning: when the days outside the current month are not shown, we are still adding `role="gridcell"`. [However](http://rawgit.com/w3c/aria/master/aria/aria.html#gridcell):

> A gridcell may be focusable, editable, and selectable.

which is clearly not the case – removing the `role` attribute won't make react-a11y warn again. Thanks for reporting this!

**Comment by gpbl** [comment]
(the fix will be published to the upcoming release)

**Comment by reintroducing** [comment]
Great, thanks for the quick turn around, really appreciate your work on this component. It's fantastic and I use it in multiple projects (not all needing this level of accessibility) :).

**Comment by gpbl** [comment]
Thanks @reintroducing for the nice words! The change has been published as v6.0.0 ([changelog](https://github.com/gpbl/react-day-picker/blob/master/CHANGELOG.md)) 🎉
